admin/app/routes/login.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+import UnauthenticatedRouteMixin from 'ember-simple-auth/mixins/unauthenticated-route-mixin';
+
+export default class LoginRoute extends Route.extend(UnauthenticatedRouteMixin) {
+
+  routeIfAlreadyAuthenticated = 'authenticated';
```

Comment:
Selon la doc, le param routeIfAlreadyAuthenticated renvoie vers l'index. Je me demande si c'est pas la page d'index qu'il faut sécuriser à l'image de mon-pix ?

routeIfAlreadyAuthenticated: String
Default: 'index'

The route to transition to if a route that implements the UnauthenticatedRouteMixin is accessed when the session is authenticated.